examples_tests/51.WavesSimulation/.gitignore

```diff
@@ -0,0 +1 @@
+bin/
```

Comment:
could we make it work in top level for everything of the form `examples_tests/*/bin/` ?